lessons/Data-Science/data-cycle/langs/en-us/index.adoc

```diff
@@ -182,77 +265,122 @@ If we want to know which cat is the heaviest, we _only care about rows for cats_
+- Return to @printable-exercise{which-question-type.adoc} For each question, determine:
+  * Which rows would you need to answer them? __(Sometimes we need all rows, and sometimes we only need a subset.)__ 
+  * Which columns would you look at? 
+  * Write your answers in the last two columns of the table at the bottom.
 - Complete @printable-exercise{which-rows-which-columns.adoc}.
 }
 
+@teacher{
+Have students share their answers and discuss any questions they have about these pages.
+}
 
 === Common Misconceptions
 
-Questions like "Who is the oldest?" or "What is the most?" require looking at _every row_ in the table.
+Questions like "Who is the oldest?" or "What is the most?" require looking at __every row__ in the table.
 
 === Synthesize
-Have students share their answers and discuss any questions they have about these pages.
 
 How does asking "Which rows? Which columns?" help us figure out @ifproglang{pyret}{what code to write}@ifproglang{codap}{which configurations to use}?
 
-== Analyzing Data @duration{15 minutes}
+
+
+== Data Cycle Practice @duration{15 minutes}
 
 === Overview
-Students progress to the third step in the Data Cycle, by combining the "Consider Data" step with their knowledge of Contracts to help them _Analyze_ that data.
+Students are introduced to the Data Cycle Pages they will be working with for the remainder of Bootstrap:Data Science.
 
-Note that @opt-printable-exercise{2-blank-data-cycles.adoc, blank Data Cycle worksheets} are available for printing as-needed!
+@teacher{If you'd like to start your students with a mini version of the Data Cycle, have them complete @opt-printable-exercise{analyzing-with-count.adoc}@ifproglang{pyret}{
+and test their code in Pyret}. 
+}
 
 === Launch
 
-@right{@image{images/AnalyzeData.png, 200}} Once we know what data we need, we can turn our attention to what we want to build with it!
+Throughout the remainder of Bootstrap:Data Science we will be using Data Cycle pages to help us answer our questions and tell our data stories. 
 
-- Do we need to filter out certain rows and make a new table?
-- Do we need a bar chart? @ifproglang{pyret}{A pie chart?} A scatter plot?
+@ifproglang{pyret}{
+@lesson-instruction{
+- Turn to @printable-exercise{data-cycle-fixed.adoc}.
+- Notice that: 
+  * the directions at the top tell you what kind of display you are going to make.
+  * there is a box in the top right corner where you will circle what kind of question is being asked.
+  * the first data cycle has already filled in which rows and columns you'll need, but you'll have to fill that in yourself for the second data cycle.
+  * there is a fill in the blank sentence in the interpret section as well as room for you to write some questions
+- Does anyone have any questions about what they're supposed to do on this page?
```

Comment:
@schanzer The reason I added that question here (and not elsewhere) is:

1.  they are going to see this format over and over throughout the course. 
2. y'all moved this page because you didn't think it was intuitive enough to engage with without explicit instruction. 

Makes sense. Not super attached.

Reply:
I reserve the right to bug you about this next year, but I'm fine with this going out for 2024